mmv1/third_party/terraform/services/modelarmorglobal/resource_model_armor_floorsetting_test.go

```diff
@@ -0,0 +1,167 @@
+package modelarmorglobal_test
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccModelArmorGlobalFloorsetting_basic(t *testing.T) {
+
+	basicContext := map[string]interface{}{
+		"location": "global",
+		"parent":   "projects/ci-test-project-188019",
```

Comment:
```suggestion
		"project_id":              envvar.GetTestProjectFromEnv(),
```

Since tests are running in different projects for presubmit and nightly